src/components/Button.js

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ buttonName }) => <button type="button">{buttonName}</button>;
+const Button = ({ buttonName, color, wide }) => (
+  <button type="button" className={`button border ${wide} ${color}`}>
+    {buttonName}
+  </button>
+);
 
 Button.propTypes = {
   buttonName: PropTypes.string,
+  color: PropTypes.string,
+  wide: PropTypes.string,
```

Comment:
As said previously, `wide` should not be a string

Reply:
I'll work on this